src/lime/graphics/Renderer.hx

```diff
@@ -30,15 +30,17 @@ class Renderer {
 	private function createGLContext(failIfMajorPerformanceCaveat:Bool):Void {
 		var transparentBackground = Reflect.hasField(window.config, "background") && window.config.background == null;
 		var colorDepth = Reflect.hasField(window.config, "colorDepth") ? window.config.colorDepth : 16;
+		var alpha = Reflect.hasField(window.config, "alpha") ? window.config.alpha : false;
 
 		var options = {
-			alpha: (transparentBackground || colorDepth > 16) ? true : false,
+			alpha: (alpha || transparentBackground || colorDepth > 16) ? true : false,
 			antialias: Reflect.hasField(window.config, "antialiasing") ? window.config.antialiasing > 0 : false,
 			depth: Reflect.hasField(window.config, "depthBuffer") ? window.config.depthBuffer : true,
 			premultipliedAlpha: true,
 			stencil: Reflect.hasField(window.config, "stencilBuffer") ? window.config.stencilBuffer : false,
 			preserveDrawingBuffer: false,
-			failIfMajorPerformanceCaveat: failIfMajorPerformanceCaveat
+			failIfMajorPerformanceCaveat: failIfMajorPerformanceCaveat,
+			powerPreference: Reflect.hasField(window.config, "powerPreference") ? window.config.powerPreference.toString() : PowerPreference.DEFAULT.toString(),
```

Comment:
I don't think you really need this `toString` here, as `getContext` options are untyped and `PowerPreference` is a String underneath anyway. If you still feel like having it, maybe at least make it `inline`?